cutten-web/src/components/AnalyzeButton.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import {
+  Button
+} from 'kingsbury/lib';
+
+class AnalyzeButton extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isClicked: false,
+    }
+  }
+
+  onClick = () => {
+    this.setState({isClicked: true})
```

Comment:
Why do we care if this button has been clicked? get rid of that bretts a dev thing